internal/server/service.go

```diff
@@ -128,6 +128,10 @@ func (s *Server) CreateEntry(ctx context.Context, req *pb.CreateEntryRequest) (*
 	if errors.Is(err, ttessera.ErrPushback) {
 		return nil, status.Errorf(codes.Unavailable, "reached max pushback; retry")
 	}
+	if errors.Is(err, context.Canceled) {
+		// Returns a 499 Client Closed Request
+		return nil, status.Error(codes.Canceled, err.Error())
+	}
 	if errors.As(err, &tessera.DuplicateError{}) {
```

Comment:
just curious about something here, as we're mixing  `errors.Is` and `errors.As`. Why do we need to use `As` if we're not further using the typed converted error, could these all just be `Is`?

Reply:
Good catch, agreed we can use `As`. Updated!